app/views/about.html

```diff
@@ -29,35 +29,12 @@ <h2 id="version">Version</h2>
                         <dd>{{version.master.kubernetes || 'unknown'}}</dd>
                       </dl>
 
-                      <h2 id="cli">Command Line Tools</h2>
-                      <p>
-                        With the OpenShift command line interface (CLI), you can create applications and manage OpenShift projects from a terminal.
-                        <span ng-if="cliDownloadURLPresent">
-                          You can download the <code>oc</code> client tool using the links below. For more information about downloading and installing it, please refer to the <a href="{{'get_started_cli' | helpLink}}">Get Started with the CLI</a> documentation.
-                        </span>
-                        <span ng-if="!cliDownloadURLPresent">
-                          Refer to the <a href="{{'get_started_cli' | helpLink}}">Get Started with the CLI</a> documentation for instructions about downloading and installing the <code>oc</code> client tool.
-                        </span>
-                        <div ng-if="cliDownloadURLPresent">
-                          <label class="cli-download-label">Download <code>oc</code>:</label>
-                          <div ng-repeat="(key, value) in cliDownloadURL">
-                            <a href="{{value}}" class="cli-download-link">
-                              {{key}}
-                              <i class="fa fa-external-link"></i>
-                            </a>
-                          </div>
-                        </div>
-                      </p>
-                      <p>
-                        After downloading and installing it, you can start by logging in using<span ng-if="sessionToken"> this current session token</span>:
-                        <div class="code prettyprint ng-binding" ng-if="sessionToken">
-                          oc login {{loginBaseURL}} --token=<span ng-show="showSessionToken">{{sessionToken}}</span><a href="#" ng-click="showSessionToken = !showSessionToken" ng-show="!showSessionToken">...click to show token...</a>
-                        </div>
-                        <pre class="code prettyprint ng-binding" ng-if="!sessionToken">
-        oc login {{loginBaseURL}}
-        </pre>
+                      <p>Refer to the OpenShift <a target="_blank" href="{{'welcome' | helpLink}}">documentation</a> for more information on how to setup, run and manage it.</p>
```

Comment:
Let's paraphrase from the docs here :) how about something like

The \<documentation\> contains information and guides to help you learn about OpenShift and start exploring its features.  From getting started with creating your first application, to trying out more advanced build and deployment techniques, it provides what you need to set up and manage your OpenShift environment as an application developer.